src/dialogs/examples/basic/PopupSignInDialog.ts

```diff
@@ -0,0 +1,35 @@
+import * as builder from "botbuilder";
+import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
+import { DialogIds } from "../../../utils/DialogIds";
+import { DialogMatches } from "../../../utils/DialogMatches";
+import { Strings } from "../../../locale/locale";
+import * as config from "config";
+
+// Demonstrates using a signin action to show a login page in a popup
+export class PopupSignInDialog extends TriggerActionDialog {
+
+    private static async popupsignin(session: builder.Session, args?: any | builder.IDialogResult<any>, next?: (args?: builder.IDialogResult<any>) => void): Promise<void> {
```

Comment:
Change name of function to sendPopUpSignInCard

Reply:
Done.